### ISSUE TITLE
Image instead of pdf preview for publications?

### ISSUE THREAD

**Issue by lordgrilo** [nplresearch/nplresearch.github.io#1]
right now the pdf preview seems to be not working for most of the papers added. I don't know if it is due to how I added the pdf/links. 
I was wondering though if it's worth putting in images instead, rather than waiting for the pdf review to come out.

**Comment by antonio-leitao** [comment]
Yeah that would be great. Ill definetely work on that but just to be sure the main idea would be?:
- The displayed image is still the first page of the PDF
- Clicking on the image leads directly to the pdf line, if any(to be implemented)

The main reason why this wasnt implemented from the start is because it will need a separate script to run at deployement that captures the pdf from the url, and saves the first page. This is not easy to do but sometimes the pdf might not be accessible through webcrawlers (openReview im looking at you), in that situation the image would have to be uploaded manually

**Comment by lordgrilo** [comment]
I think it would be cool to just put an image (which could be the first page of the pdf or something else, like a plot from the paper, or something midjourney generated? ;) )
In this way we would have more freedom to put stuff, rather than be limited to the pdf page, and also it's probably easier to just drop an image in the folder than run the script to capture the first page of the pdf.
